buildSrc/subprojects/jvm/jvm.gradle.kts

```diff
@@ -2,6 +2,8 @@ dependencies {
     implementation(project(":basics"))
     implementation(project(":dependencyModules"))
     implementation(project(":moduleIdentity"))
+    implementation(project(":performanceTesting"))
+    implementation(project(":integrationTesting"))
```

Comment:
:o: I would like us to not re-introduce these dependencies to keep things as separated as we can. In particular in light of https://github.com/gradle/gradle-private/issues/3139.

I know this further down is a bit ugly: `this.javaClass.simpleName != "PerformanceTest"`
But we should solve that differently in the context of https://github.com/gradle/gradle-private/issues/3126 or https://github.com/gradle/gradle-private/issues/3136

Reply:
👍